sql/moz-fx-data-shared-prod/telemetry_derived/clients_last_seen_by_default_browser_profile_age_v1/query.sql

```diff
@@ -0,0 +1,24 @@
+SELECT
+  submission_date,
+  is_default_browser,
+  CASE
+    WHEN days_since_first_seen < 28
+      THEN TRUE
+    ELSE FALSE
+  END AS is_new_profile,
+  normalized_os_version,
+  os,
+  normalized_channel,
+  COUNT(*) AS row_count,
```

Comment:
I also suggest adding country. This shows up for example in the health indicators dashboard.